stage1_fly/run/main.go

```diff
@@ -219,7 +219,15 @@ func stage1() int {
 	}
 
 	// TODO: insert environment from manifest
-	env := []string{"PATH=/bin:/sbin:/usr/bin:/usr/local/bin"}
+	defaultEnvPath := "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+	envList := p.Manifest.Apps[0].App.Environment
+	env := make([]string, len(envList)+1)
```

Comment:
This isn't right: http://play.golang.org/p/PxKbFYQoEG
I think this can all be greatly simplified

```
        env := []string{"PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"}
	for _, e := range p.Manifest.Apps[0].App.Environment {
		env = append(env, e.Name+"="+e.Value)
	}
```